packages/roosterjs-editor-dom/lib/clipboard/extractClipboardEvent.ts

```diff
@@ -27,22 +27,23 @@ export default function extractClipboardEvent(
     event: ClipboardEvent,
     callback: (clipboardData: ClipboardData) => void,
     options?: ExtractClipboardEventOption,
-    rangeBeforePaste?: Range
+    rangeBeforePaste?: Range,
+    pasteNativeEvent: boolean = false
```

Comment:
When there is ClipboardEvent, it must be from native paste, so no need to provide this parameter here, and always pass true